ray-operator/config/samples/ray-cluster.auth.yaml

```diff
@@ -79,7 +79,7 @@ spec:
               command:
               - bash
               - -c
-              - wget -T 2 -q -O- http://localhost:52365/api/local_raylet_healthz | grep success && wget -T 10 -q -O- http://localhost:8443/api/gcs_healthz | grep success
+              - wget -T 10 -q -O- http://localhost:52365/api/local_raylet_healthz | grep success && wget -T 10 -q -O- http://localhost:8443/api/gcs_healthz | grep success
```

Comment:
it's not enough to increase wget timeout, you need to also increase `probeTimeout` in the container's probe config